openassessment/workflow/models.py

```diff
@@ -352,24 +356,74 @@ def _get_steps(self):
             steps = list(self.steps.all())
         return steps
 
-    def set_score(self, score):
+    def set_staff_score(self, score, staff_name, is_required=False, is_override=False, reason=None):
```

Comment:
Good point. I suppose we'll never have neither (if it's not required or an override, what is it?) or both, so we can assume one and treat "not that" as the other.